data/json/terrain.json

```diff
@@ -2980,6 +2998,7 @@
         "type" : "terrain",
         "id" : "t_rock_smooth",
         "name": "smoothed rock",
+        "description": "Shaped naturally occurring stonework, commonly granite or marble for older structures.",
```

Comment:
I'm not entirely sure what this means...


> Shaped naturally occurring stonework, commonly granite or marble for older structures

Should the first part read something along the lines of, "A block of stone that's been smoothed and shaped"?

And what does the second part mean? Smoothed granite and marble are more common in older structures than other types of smoothed rock?

Reply:
This is actually a stilted sentence, that I should've rewrote. However, this terrain is referenced only in two very different places: [basements](https://github.com/CleverRaven/Cataclysm-DDA/blob/93f2979292e8e0082ea32f6a9720ef51d810b8e8/src/mapgen_functions.cpp#L2932) and [cemeteries](https://github.com/CleverRaven/Cataclysm-DDA/blob/3f93d321a11aae1850cbfeabc2314ff08bffbcb2/data/json/mapgen/cemetery_4square.json#L91). Hence one of the reasons I write in the PR intro that trying to write some of these descriptions can be very broad.

So for whatever reason, the cemetery looks like where they hold a wake or funeral but in a building of smoothed rock? So, I researched what type of buildings like that would be found in New England region and granite/marble were the most common. The first part is meant more for the basements.

I think your `A block of stone that's been smoothed and shaped` is a good start that I'll work off of.